subiquity/common/filesystem/manipulator.py

```diff
@@ -197,9 +197,10 @@ def reformat(self, disk):
             self.delete_partition(p, True)
         self.clear(disk)
 
-    def partition_disk_handler(self, disk, partition, spec):
-        log.debug('partition_disk_handler: %s %s %s', disk, partition, spec)
-        log.debug('disk.freespace: {}'.format(gaps.largest_gap_size(disk)))
+    def partition_disk_handler(self, disk, spec, *, partition=None, gap=None):
+        log.debug('partition_disk_handler: %s %s %s %s',
+                  disk, spec, partition, gap)
+        log.debug('disk.freespace: %s', disk.available_for_partitions)
```

Comment:
Not quite sure why you'd change this back to disk.available_for_partitions, I thought we wanted to kill that? Maybe just drop the log message I guess...

Reply:
Yes, the available value was the only reasonable alternative.  Dropping the log statement(s).